test_data/yast/encryption/encrypt_no_lvm.yaml

```diff
@@ -1,3 +1,3 @@
 crypttab:
   num_devices_encrypted: 3
-$include: test_data/yast/encryption/default_enc.yaml
+<<: !include test_data/yast/encryption/default_enc.yaml
```

Comment:
I would check this one as we have 2 includes here. Did you check if structure we receive is valid? Maybe we should get rid of this file, and move that single setting to the schedule instead.

Reply:
I checked this particular case, as it is the only one usage, and it works correctly.

I even provided the VR for the case: https://openqa.suse.de/tests/4275923.

I didn't research the reason why it was done like that, but it works as before. Also, it seems to be using with YAML_TEST_DATA variable. So, this might be related to the way how the test data is loaded for this particular test.

Maybe we should get rid of this file or refactor it somehow, but I would leave this discussion for another topic.

Reply:
Sure. If it works we are good. It just looks sub optimal to have chain of files here with a single setting. Could you please file a progress ticket, so we don't forget?